tests/unit/Jadu/Pulsar/Twig/Macro/Fixtures/tabs/tabs_list-disabled.html

```diff
@@ -1,7 +1,7 @@
 <div class="nav-inline t-nav-inline">
     <ul class="nav-inline__list">
         <li class="nav-inline__item nav-inline__item--is-active is-active">
-            <a href="#" data-toggle="link" aria-disabled="true" class="nav-inline__link is-disabled">bar</a>
+            <a href="#" data-toggle="link" aria-disabled="true" tabindex="-1" class="nav-inline__link is-disabled">bar</a>
```

Comment:
Pretty sure it does but just to double check - `is-disabled` will prevent mouse focus won't it? Otherwise, even though it has tabindex="-1" it could still be focused by mouse.

Reply:
Yes.